src/js/player.js

```diff
@@ -3684,6 +3688,14 @@ class Player extends Component {
     } else if (!value) {
       this.options_.autoplay = false;
 
+    // normalize `true` as 'play' if need be
+    } else if (value === true && this.options_.normalizeAutoplay) {
```

Comment:
Should this check be combined into the if on line 3681 above? This way we aren't duplicated the lines below.